CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## 0.14.4
 ###### _Feb 02, 2016_
 
+##### Breaking Changes
+- [FloatingActionButton] Fixed a className issue 
```

Comment:
That won't help people to upgrade. What about:
`The className is now applied on the root component`